libc/test/src/__support/HashTable/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+add_libc_test(
+  bitmask_test
+  SUITE
+    libc-support-tests
+  SRCS
+    bitmask_test.cpp
+  DEPENDS
+    libc.src.__support.HashTable.bitmask
+    libc.include.stdlib
+)
+add_libc_test(
+  table_test
+  SUITE
+    libc-support-tests
+  SRCS
+    table_test.cpp
+  DEPENDS
+    libc.src.__support.HashTable.randomness
+    libc.src.__support.HashTable.table
+    libc.src.__support.common
+  UNIT_TEST_ONLY
+)
+cpu_supports(CPU_SUPPORTS_SSE2 "SSE2")
+if (CPU_SUPPORTS_SSE2)
```

Comment:
I understand that there is a similar function defined in `string`. However, it is not exposed and it dispatches targets into a complicated feature matrix.

Reply:
You can add automated testing for SSE2 as follow:
- define `SSE2_OPT` similar to `ROUND_OPT` flag in `libc/cmake/modules/LLVMLibCFlagRules.cmake`
- expand `${SSE2_OPT_FLAG}` similar to `${ROUND_OPT_FLAG}` in `libc/cmake/modules/LLVMLibCObjectRules.cmake`, function `_get_common_compile_options`
- add `FLAGS SSE2_OPT` inside `add_header_library(bitmask ...)` in `libc/src/__support/HashTable/CMakeLists.txt` above.

Then all tests and targets depending on `libc.src.__support.HashTable.bitmask` will be extended to both with and without SSE2.